Formula/pytouhou.rb

```diff
@@ -25,7 +25,7 @@ class Pytouhou < Formula
   depends_on "gtk+3" => :recommended
   if build.with? "gtk+3"
     depends_on "py3cairo" # FIXME: didn't get picked up by pygobject3 below
-    depends_on "pygobject3" => "with-python3"
+    depends_on "pygobject3"
```

Comment:
Realizing that this won't work as I expect it to -- misread the documentation.  Will continue my journey on this tomorrow, when I'm more awake!